### PR TITLE
Add compatibility Sylius 1.11

### PR DESCRIPTION
**Pull request by xaviermarchegay**
Add the compatibility with Sylius 1.11

### CODE REVIEW

**Comment by mamazu** [comment]
Looks like we need to update the github configuration from the base repository. It seems that the node version has changed.

**Comment by xaviermarchegay** [comment]
Well, the base repository also fails so I pushed the node version from the github actions of sylius/sylius.

I also changed the symfony versions in composer.json

**Comment by mamazu** [comment]
Hm, tests are failing but I think that is because of something else. Can you try to debug them?

**Comment by xaviermarchegay** [comment]
I fixed a number of errors (exclude some testing combinations, fixed the .env, removed a unsupported node version, etc.)

The remaining error looks like: https://gitlab.com/DMore/chrome-mink-driver/-/issues/119

I am going to try to reproduce the error on my env

**Comment by mamazu** [comment]
If you can't reproduce them, then we will just check it again manually and then merge it without the pipeline being green.

**Comment by xaviermarchegay** [comment]
I tried different things but I cannot seem to reproduce it...

**Comment by mamazu** [comment]
Yeah, behat can be a pain in the butt sometimes. We'll just merge it without it being green then.

File: tests/Application/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "chart.js": "^2.9.3",
-    "jquery": "^3.4.0",
```
Comment (mamazu) [inline tests/Application/package.json:5]: Is this backwards compatible? Otherwise we also have to find a way to only enable it conditionally.

Comment (xaviermarchegay) [reply tests/Application/package.json]: I have no idea why I upgraded jquery, the Sylius plugin skeleton still has ^3.4.0
Anyway, I reverted the change.

**Pull request merged**